R/uvhydrograph-render.R

```diff
@@ -167,6 +167,72 @@ createSecondaryPlot <- function(data, month){
       status_msg <- paste('Corrected data missing for', data$reportMetadata$secondaryParameter)
     }
   } 
+
+  isUpchainSeries <- any(grepl("downsampledUpchainSeries", names(data)))
```

Comment:
Yeah I was thinking that initially but PT thought I should use different variable names for each set of data (I.E: "corr_UV_ref" and "corr_UV_up" in place of "corr_UV2"). So then the calls for stuff like YAxisInterval need to vary based on whether we're looking at reference or upchain data because the variables we use in the call change (such as "secondaryData$corr_UV_ref" vs "secondaryData$corr_UV_up") so I wasn't sure what the best way to handle that would be aside from separating it out like this?